website/content/usage/_index.md

```diff
@@ -20,6 +20,11 @@ address, or if the address is already in use by another service,
 assignment will fail and MetalLB will log a warning event visible in
 `kubectl describe service <service name>`.
 
+The `spec.loadBalancerIP` parameter is going to be deprecated soon, so
```

Comment:
I'd reword this to:

"We both support `spec.loadBalancerIP` and a custom `metallb.universe.tf/loadBalancerIPs`.
`metallb.universe.tf/loadBalancerIPs` also supporst a comma separated list of IPs to be used in case of Dual Stack services.

Please note that `spec.LoadBalancerIP` is planned to be deprecated in [k8s apis](https://github.com/kubernetes/kubernetes/pull/107235).
"

Reply:
done